core/templates/components/skill-selector/merge-skill-modal.controller.spec.ts

```diff
@@ -41,14 +41,15 @@ describe('Merge Skill Modal Controller', function() {
     });
   }));
 
-  it('should check properties set after controller is initialized', function() {
+  it('should initialize correctly $scope properties after controller' +
+    ' initialization', function() {
     expect($scope.skillSummaries).toBe(skillSummaries);
     expect($scope.categorizedSkills).toBe(categorizedSkills);
     expect($scope.allowSkillsFromOtherTopics).toBe(true);
     expect($scope.selectedSkillId).toBe('');
   });
 
-  it('should close modal on saving', function() {
+  it('should close modal on save', function() {
```

Comment:
This is not very understandable. What is being saved? When is save() called? Better to qualify "save" better

Reply:
Done.